src/auth.service.js

```diff
@@ -21,7 +19,7 @@
     var eventBus = $rootScope.$new(true);
 
     function getSession() {
-      var sessionUrl = options.sessionEndpoint;
+      var sessionUrl = options.url + options.sessionEndpoint;
```

Comment:
if you remove the `url` in the docs, you should remove it here too, shouldn't you? I.e. leave it as it is.